ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -79,6 +79,65 @@ module('Acceptance | secrets/secret/create', function(hooks) {
     assert.ok(showPage.editIsPresent, 'shows the edit button');
   });
 
+  // https://github.com/hashicorp/vault/issues/5960
+  test('version 1: nested paths creation maintains ability to navigate the tree', async function(assert) {
+    let enginePath = `kv-${new Date().getTime()}`;
+    let secretPath = '1/2/3/4';
+    // mount version 1 engine
+    await mountSecrets.visit();
+    await mountSecrets.selectType('kv');
+    await withFlash(
+      mountSecrets
+        .next()
+        .path(enginePath)
+        .version(1)
+        .submit()
+    );
+
+    await listPage.create();
+    await editPage.createSecret(secretPath, 'foo', 'bar');
+
+    // setup an ancestor for when we delete
+    await listPage.visitRoot({ backend: enginePath });
+    await listPage.secrets.filterBy('text', '1/')[0].click();
+    await listPage.create();
+    await editPage.createSecret('1/2', 'foo', 'bar');
+
+    // lol we have to do this because ember-cli-page-object doesn't like *'s in visitable
```

Comment:
Ha yep, that was exactly the issue I was having - for sure going to steal this later, but probably just leave as-is for now. 

I see you added the ability to pass an array of paths too - how do you determine which one to visit when you do that?

Reply:
Cool, I owe you an aws icon anyway 😂 

If you create the visitable with 2 paths, say:

```javascript
visitable(['/:dc/kv/:kv', '/:dc/kv'], str => str)
```

.. and you only you don't provide all the keys when you try to visit it, something like:

```javascript
visit(
  {
    dc: 'dc-1'
  }
);
```

then it will use the one that it has complete data for.

So that above visit call will use the second array item for the URL in the above code, and:

```javascript
visit(
  {
    dc: 'dc-1',
    kv: 'some/kv/deep/down' 
  }
);
```

...would use the first one. Not sure if I've explained that well enough, ping me if you like if it's not clear.

I think (as far as I remember) I made it backwards compatible with the original implementation, so you can still just pass it a string like the original one.

Reply:
Cool np, yeah in this case if I want the index page, I don't pass it a KV, whereas if I want to test an actual KV page I pass it a KV. Actually I'm just remembering now, I think I use it a lot for the 'create or edit' thing. If I pass a 'slug' it means I want an edit page, whereas if I don't I want a 'create'. It's almost like how the router works I suppose.